dataset/hotspotArea.txt

```diff
@@ -3316,3 +3270,182 @@ Haryana	Faridabad	Dabua Colony	https://www.facebook.com/ssffaridabad/posts/89812
+Telangana	Hyderabad	Asif Nagar; Venkatamma thota, Feel Khana, Asifnagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Asif Nagar; Opp Mohammadiya Masjid, Gudi Malkapur	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Tappachabutra; Dargah Ganja Shahi	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Tappachabutra; Asifnagar Zirrah	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jubilee Hills; Beside Anganwadi Center, Venkatagiri Hylam Colony Area	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Banjara Hills; IAS Colony, Shaikpet	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Banjara Hills; Hakeempet Tolichowki	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Banjara Hills; Filmy Nagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Banjara Hills; Syed Nagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Banjara Hills; Shoukath Nagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward no. 133,Chandranagar, Yaseen House Souroundings, Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ameem House Souroundings, Rajeev gandhi nagar ward no 135 Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward no. 134, Rashhed khan House Souroundings, Hasmathpet Rajeev gandhi nagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward no 135, Syed Ilias House Souroundings, Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward no 134, Md. Khayuum House Souroundings,Temple Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward No. 119, Md. Razeeyuddin House Souroundings,Hasmathpet, Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Alwal; Ward no. 119, Abdul Hafeez House, Souroundings,Hasmathpet Alwal	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jagadgirigutta; Chandragiri nagar	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jagadgirigutta; Yellammabanda, Jagathgirigutta	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jagadgirigutta; Modi apts, block-b, apuroopa colony	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jagadgirigutta; Khaja House Souroundings, Apuroopa colony	https://twitter.com/Sumiran_KVS/status/1256644111060230144
+Telangana	Hyderabad	Jagadgirigutta; Wazeed House Souroundings, Subhash Nagar, Jeedimetla	https://twitter.com/Sumiran_KVS/status/1256644111060230144
```

Comment:
can't merge above lines as resident names are given in the list

Reply:
fixed in the d48a118cc0da8140b88157220fff7910be67e36d commit